pyro/distributions/__init__.py

```diff
@@ -2,14 +2,12 @@
 
 from pyro.distributions.delta import Delta
 from pyro.distributions.distribution import Distribution
+from pyro.distributions.omt_mvn import OMTMultivariateNormal
 from pyro.distributions.rejector import Rejector
-from pyro.distributions.torch import (Bernoulli, Beta, Binomial, Categorical, Cauchy, Dirichlet, Exponential, Gamma,
-                                      LogNormal, Multinomial, Normal, OneHotCategorical, Poisson,
-                                      TransformedDistribution, Uniform)
+from pyro.distributions.torch import *
```

Comment:
didnt you just berate me a few days ago to never `import *`!

Reply:
This is the exception that makes the rule :wink: In pyro.distributions.torch we are dynamically populating the module, therefore explicit import is impossible. Instead we do the next best: we explicitly construct the `__all__` list.